src/Microsoft.Data.SqlClient/add-ons/AzureKeyVaultProvider/Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider.csproj

```
@@ -15,6 +15,14 @@
     <BuildProjectReferences>false</BuildProjectReferences>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+  </PropertyGroup>
+   <!--Generating Strong Name-->
+  <PropertyGroup Condition="$(CDP_BUILD_TYPE)==Official">
```

Normally string comparisons in msbuild should use quotes, or is this something else?

```suggestion
  <PropertyGroup Condition="'$(CDP_BUILD_TYPE)'=='Official'">
```